### PR TITLE
fontforge 20150228

### PR DESCRIPTION
**Pull request by DomT4**
I’m an idiot.

### CODE REVIEW

File: Library/Formula/fontforge.rb [hunk 1]
```diff
@@ -1,7 +1,9 @@
 class Fontforge < Formula
   homepage "https://fontforge.github.io"
+  # We use the archive tarball because it's smaller and compiles fine.
```
Comment (adamv) [inline Library/Formula/fontforge.rb]: Smaller than what?

Comment (mikemcquaid) [inline Library/Formula/fontforge.rb]: Can probably omit this comment.

Comment (DomT4) [inline Library/Formula/fontforge.rb]: @adamv The 255MB "release" tarball. Talked to upstream about this pretty extensively in the past and more recently, and their opinion essentially is if we can use the smaller tarball and end up with the same end-result, which we do, saving users from a >250MB download is only a good thing :smile_cat:.

@mikemcquaid Will chop it.

**Comment by DomT4** [comment]
Updated PR for new upstream version release. Upstream have tagged it pre-release because they don't consider it officially available to users until there are binaries available, and until it lands in Homebrew, but they've confirmed this is an official release we can carry.

**Comment by mikemcquaid** [comment]
Thanks @DomT4!

**Pull request closed without merging**